__tests__/auth-provider.test.ts

```
@@ -405,7 +405,7 @@ describe('AuthProvider', () => {
 			const tamperedHeader = Buffer.from('tampered_header').toString('base64');
 
 			// Tamper the Header
-			const tampered = serialized.replace(/.*:/g, `${tamperedHeader}:`);
+			const tampered = serialized!.replace(/.*:/g, `${tamperedHeader}:`);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)